sky/shell/platform/ios/semantics_listener_impl.mm

```diff
@@ -0,0 +1,245 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "sky/shell/platform/ios/semantics_listener_impl.h"
+
+namespace sky {
+namespace shell {
+
+SemanticsListenerImpl::SemanticsListenerImpl(
+    FlutterView* view,
+    semantics::SemanticsServerPtr semanticsServer)
+    : _view(view), _binding(this) {
+  mojo::InterfaceHandle<semantics::SemanticsListener> listener;
+  _binding.Bind(&listener);
+  semanticsServer->AddSemanticsListener(listener.Pass());
+}
+
+void SemanticsListenerImpl::UpdateSemanticsTree(
+    mojo::Array<semantics::SemanticsNodePtr> nodes) {
+  for (const semantics::SemanticsNodePtr& node : nodes) {
+    updateSemanticsNode(node);
+  }
+  NSArray* accessibleElements = createAccessibleElements();
```

Comment:
> If you make this use an @property then this becomes
_view.accessibleElements = createAccessibleElements();

You would still need to release the results of `createAccesibleElements` since that method has transfer-out ownership on the return.